JavaScript/Topics/3.Web_Browser/README.md

```diff
@@ -1 +1,124 @@
-# To Do
\ No newline at end of file
+# JavaScript's usage in the Web Browser
+
+JavaScript is commonly utilised in both front-end and back-end web development. However, the focus of this lesson will be on front-end JavaScript implementation.
+
+## Where Javascript is used in Front-End?
+
+The structure of a web page is provided by HTML, the layout and rudimentary animations are provided by CSS, and the website's interactivity is provided by JavaScript.
+
+JavaScript can support:<br>
+
+1. Rich UI libraries
+2. Validation of forms, which may be linked to the backend
+3. Browser APIs to manipulate HTML and CSS
+4. Three-dimensional animations which CSS isn't capable
+   
+Plus a lot more!
+
+The following material shows how to use the `<script>` element in a web browser to implement JavaScript.
+<hr>
+<br>
+
+# Use of `<script>` tag
+
+Externally, JavaScript may be used by generating a separate file with the `.js` extension and linking to the HTML file. This is called *external JavaScript*. Or it can be used inside HTML files which is called *internal JavaScript*.
```

Comment:
```suggestion
Externally, JavaScript can be used by generating a separate file with the `.js` extension and linking to the HTML file. This is called *external JavaScript*. Or it can be used inside HTML files which is called *internal JavaScript*.
```